src/components/templates/nav-item/styles.ts

```diff
@@ -7,6 +7,22 @@ export const useStyles = tss.create(({ theme }) => {
       alignItems: 'center',
       gap: '10px',
       cursor: 'pointer',
+      width: '256px',
+      height: '40px',
+      paddingLeft: '15px',
+      paddingRight: '10px',
+      '&:hover': {
```

Comment:
Move hover to the end

Reply:
Hover is already at the bottom of navItem style.